src/Test/L0/Listener/AgentL0.cs

```diff
@@ -573,5 +573,121 @@ public async void TestRunOnceHandleUpdateMessage()
                 _messageListener.Verify(x => x.DeleteMessageAsync(It.IsAny<TaskAgentMessage>()), Times.Once());
             }
         }
+
+        [Theory]
+        [Trait("Level", "L0")]
+        [Trait("Category", "Agent")]
+        [InlineData("--help")]
+        [InlineData("--version")]
+        [InlineData("--commit")]
+        public async void TestInfoArgumentsCLI(string arg, int expected=Constants.Agent.ReturnCode.Success)
```

Comment:
Should we add one that fails?

Reply:
that's a good point, I'm not sure if you can do that with a Theory, I'll take a look